.changeset/angry-penguins-flow.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sap-cloud-sdk/connectivity': minor
+'@sap-cloud-sdk/test-util': minor
+'@sap-cloud-sdk/integration-tests': minor
```

Comment:
We will not release this package, so we should not add this?

Reply:
also removed